Bicep.sln

```diff
@@ -1,163 +1,163 @@
-		{34D7A843-55CB-4A3A-B268-2D4ABB7044D3}.Release|Any CPU.Build.0 = Release|Any CPU
-		{4CD48B4A-6297-49FC-90AC-5DF9213C1527}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
-		{4CD48B4A-6297-49FC-90AC-5DF9213C1527}.Debug|Any CPU.Build.0 = Debug|Any CPU
-		{4CD48B4A-6297-49FC-90AC-5DF9213C1527}.Release|Any CPU.ActiveCfg = Release|Any CPU
-		{4CD48B4A-6297-49FC-90AC-5DF9213C1527}.Release|Any CPU.Build.0 = Release|Any CPU
-		{695560CE-C6C3-4FCD-A488-09F06E608297}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
-		{695560CE-C6C3-4FCD-A488-09F06E608297}.Debug|Any CPU.Build.0 = Debug|Any CPU
-		{695560CE-C6C3-4FCD-A488-09F06E608297}.Release|Any CPU.ActiveCfg = Release|Any CPU
-		{695560CE-C6C3-4FCD-A488-09F06E608297}.Release|Any CPU.Build.0 = Release|Any CPU
-		{F3AF01F6-24E8-4129-80B6-84AC070B5C7D}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
-		{F3AF01F6-24E8-4129-80B6-84AC070B5C7D}.Debug|Any CPU.Build.0 = Debug|Any CPU
-		{F3AF01F6-24E8-4129-80B6-84AC070B5C7D}.Release|Any CPU.ActiveCfg = Release|Any CPU
-		{F3AF01F6-24E8-4129-80B6-84AC070B5C7D}.Release|Any CPU.Build.0 = Release|Any CPU
-		{61026689-70A0-403E-B616-DFAEEF259444}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
-		{61026689-70A0-403E-B616-DFAEEF259444}.Debug|Any CPU.Build.0 = Debug|Any CPU
-		{61026689-70A0-403E-B616-DFAEEF259444}.Release|Any CPU.ActiveCfg = Release|Any CPU
-		{61026689-70A0-403E-B616-DFAEEF259444}.Release|Any CPU.Build.0 = Release|Any CPU
-	EndGlobalSection
-	GlobalSection(SolutionProperties) = preSolution
-		HideSolutionNode = FALSE
-	EndGlobalSection
-	GlobalSection(NestedProjects) = preSolution
-		{58F20140-729B-41E0-91F0-7004C4E0CB1E} = {B02251C3-B01E-40EB-B145-2B03F25FE653}
-		{6679B13D-BA2B-43C8-A548-6C6D22E19BE1} = {9EBF2BF0-C968-4029-B21D-214D195EA148}
-		{D72C8232-24ED-4EDD-ABDB-16194681D9F2} = {FE323E78-E865-46E2-859A-E4F6FB312C0F}
-		{F4AF2603-C020-4A57-A428-AF3EFED21544} = {76F6CBAF-FE81-4B56-B0DB-DE6FD7174771}
-		{087ADE61-3D39-4884-8258-40D7A7DD833F} = {FE323E78-E865-46E2-859A-E4F6FB312C0F}
-		{166B8503-54F0-452F-88C5-1430BF05A604} = {FE323E78-E865-46E2-859A-E4F6FB312C0F}
-		{F7B0F06D-56CE-4F4A-8EF9-ED6EC3D77EE7} = {76F6CBAF-FE81-4B56-B0DB-DE6FD7174771}
-		{A2E4B91C-3B7F-4CB1-B482-BE40F7709EB5} = {FE323E78-E865-46E2-859A-E4F6FB312C0F}
-		{29A1AE2B-D47D-41DD-9824-279475D9B6C2} = {B02251C3-B01E-40EB-B145-2B03F25FE653}
-		{0DE47D02-2BBE-4352-9E38-79A374F9C41E} = {76F6CBAF-FE81-4B56-B0DB-DE6FD7174771}
-		{34D7A843-55CB-4A3A-B268-2D4ABB7044D3} = {B02251C3-B01E-40EB-B145-2B03F25FE653}
-		{4CD48B4A-6297-49FC-90AC-5DF9213C1527} = {61643BA1-BE82-4430-A3D3-CB7A16E747E3}
-		{695560CE-C6C3-4FCD-A488-09F06E608297} = {61643BA1-BE82-4430-A3D3-CB7A16E747E3}
-		{F3AF01F6-24E8-4129-80B6-84AC070B5C7D} = {61643BA1-BE82-4430-A3D3-CB7A16E747E3}
-		{61026689-70A0-403E-B616-DFAEEF259444} = {38A0A11F-72BD-4512-A4A9-AC953936C09F}
-	EndGlobalSection
-	GlobalSection(ExtensibilityGlobals) = postSolution
-		SolutionGuid = {21F77282-91E7-4304-B1EF-FADFA4F39E37}
-	EndGlobalSection
-EndGlobal
+﻿

```

Comment:
... because it's trying to find bicepsettings.json in (I think) the workspace root and is not finding it.

ConfigHelper::ConfigHelper:
this.Config = this.ConfigBuilder.Build();  <<<<<<<<<<<<<<<

Exception has occurred: CLR/System.IO.FileNotFoundException
	•	
Eine Ausnahme vom Typ "System.IO.FileNotFoundException" ist in System.Private.CoreLib.dll aufgetreten, doch wurde diese im Benutzercode nicht verarbeitet.: 'The configuration file 'bicepsettings.json' was not found and is not optional. The physical path is '/Users/stephenweatherford/repos/template-examples/bicepsettings.json'.'
   bei System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   bei Microsoft.Extensions.Configuration.FileConfigurationProvider.HandleException(ExceptionDispatchInfo info)
   bei Microsoft.Extensions.Configuration.FileConfigurationProvider.Load(Boolean reload)
   bei Microsoft.Extensions.Configuration.FileConfigurationProvider.Load()
   bei Microsoft.Extensions.Configuration.ConfigurationRoot..ctor(IList`1 providers)
   bei Microsoft.Extensions.Configuration.ConfigurationBuilder.Build()
   bei Bicep.Core.Configuration.ConfigHelper..ctor() in /Users/stephenweatherford/repos/johnbicep/src/Bicep.Core/Configuration/ConfigHelper.cs: Zeile34
   bei Bicep.Core.Analyzers.Linter.LinterRuleBase..ctor(String code, String ruleName, String description, String docUri, Boolean enableForEdit, Boolean enableForCLI, DiagnosticLevel diagnosticLevel) in /Users/stephenweatherford/repos/johnbicep/src/Bicep.Core/Analyzers/Linter/LinterRuleBase.cs: Zeile24
   bei Bicep.Core.Analyzers.Linter.Rules.BCPL1000..ctor() in /Users/stephenweatherford/repos/johnbicep/src/Bicep.Core/Analyzers/Linter/Rules/BCPL1000.cs: Zeile14

Reply:
Because of:
var localConfig = Path.Join(Directory.GetCurrentDirectory(), settingsfile);

...  We already decided you'd be changing this logic, but need to make sure we only add if it exists.

Also, you should add a work item to dynamically pick up changes to the settings file (or new or deleted settings files) when in vscode.

Reply:
Default settings are moving to resource